shor/providers/qiskit/IBMQ.py

```diff
@@ -0,0 +1,20 @@
+from qiskit.providers.ibmq import IBMQ
+
+from shor.providers.qiskit.base import QiskitProvider
+
+DEFAULT_BACKEND = "qasm_simulator"
+
+
+class IBMQProvider(QiskitProvider):
+    def __init__(self, **config):
+        if not IBMQ.active_account():
+            IBMQ.load_account()
+
+        provider = IBMQ.providers()[0] if IBMQ.providers() else None
```

Comment:
I'm a little confused by this line right here. What are we expecting IBMQ.providers() to return?

Reply:
A list or None

For [ ] or None this line would evaluate to None